plugin/src/main/java/com/playmonumenta/libraryofsouls/SoulEntry.java

```diff
@@ -248,6 +265,7 @@ public void openBestiary(Player player, BestiaryArea parent, List<BestiaryEntryI
 	 *--------------------------------------------------------------------------------*/
 
 	public enum InfoTier {
+		LORE(4),
```

Comment:
I don't think we should add another tier here - would rather "Everything" includes the lore too.

Reply:
Looks like this is still here?